src/mdx-pages/guides/spatial-navigation.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Spatial Navigation
+category: advanced
+description: Explaining the ways to set up Spatial Navigation in Video.js player.
+---
+
+
+Spatial Navigation in Video.js enhances user experience and accessibility on smart TV devices. This functionality enables seamless navigation through interactive elements within the player using remote control arrow keys.
+
+## Spatial Navigation Class
+
+The Spatial Navigation Class within Video.js offers essential functionality, allowing users to effortlessly navigate through focusable components.
+
+### Methods
+
+* `start()` - Starts the Spatial Navigation system by adding keydown event listeners to the player.
+* `stop()` - Stops the Spatial Navigation by removing the keydown event listener from the player and updating the internal state.
+* `pause()` - Temporarily disables the Spatial Navigation functionality without removing the event listeners.
+* `resume()` - Re-enables the Spatial Navigation functionality after it has been paused.
+* `updateFocusableComponents()` - Updates and retrieves the list of focusable components within the player.
```

Comment:
This may need to be updated according to https://github.com/videojs/video.js/pull/8717

Reply:
Changed

Reply:
@amtins PR https://github.com/videojs/video.js/pull/8717 was removed